packages/grid_client/scripts/tft.ts

```diff
@@ -0,0 +1,28 @@
+import { substrateURL, TFT } from "../src";
+import { getClient } from "./client_loader";
+
+const substrate = "wss://tfchain.dev.grid.tf/ws" as substrateURL.DEV;
```

Comment:
why do you redefine it again?

```ts
const substrate = substrateURL.DEV;
```

Reply:
Not needed anymore. Please, check the changes.